app/dashboard/views.py

```diff
@@ -1120,7 +1121,60 @@ def profile(request, handle):
                 handle = handle[:-1]
             profile = profile_helper(handle, current_user=request.user)
 
-        context = profile.to_dict()
+        tabs = [
+            ('all', _('All Activity')),
+            ('new_bounty', _('Bounties Funded')),
+            ('start_work', _('Work Started')),
+            ('work_submitted', _('Work Submitted')),
+            ('work_done', _('Bounties Completed')),
+            ('new_tip', _('Tips Sent')),
+            ('receive_tip', _('Tips Received')),
+        ]
+        page = request.GET.get('p', None)
+
+        if page:
```

Comment:
any way we can DRY between the if/else statement here?

Reply:
Yes, possible 👍

Reply:
ok lets do it pls

Reply:
already cleaned it in this in this PR. Though I still need the `if else` 😅